### PR TITLE
Split apps

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

File: inspector/app/renderer/actions/Session.js.orig [hunk 1]
```diff
@@ -0,0 +1,793 @@
+import settings, { SAVED_SESSIONS, SERVER_ARGS, SESSION_SERVER_TYPE, SESSION_SERVER_PARAMS
```
Comment (KazuCocoa) [inline inspector/app/renderer/actions/Session.js.orig:1]: Can remove? `.orig`

**Comment by KazuCocoa** [comment]
Mm, CI on macOS claimed as below. Let me check

```
> appium-desktop@1.19.0-beta.0 build:main /Users/runner/work/1/s/server
> cross-env NODE_ENV=production parcel build --target node --public-url . --out-dir ./dist --out-file main.js ./app/main.js

events.js:170
      throw er; // Unhandled 'error' event
      ^
```

**Comment by jlipps** [comment]
I'll need to adjust CI, I didn't know where to find it before but I'll try to fix it up.

**Comment by jlipps** [comment]
OK CI is working now. and note that since i had to merge in commits from master, well need to do a merge commit to get this in rather than rebasing.

**Review by KazuCocoa — APPROVED** [review]
👍

**Comment by KazuCocoa** [comment]
I'll work for an issue, https://github.com/appium/appium-desktop/issues/1603 , after merging this.
(The issue needs to commit to webdriverio first tho)

**Comment by KazuCocoa** [comment]
the CLA's pending status is probably github bot's one

**Comment by jlipps** [comment]
Yeah I think we can ignore that check. @dpgraham would love to have at least a birds eye review to see if you like the new arrangement.

**Review by dpgraham — APPROVED** [review]
The approch looks good. Obviously this is a big PR and I don't think I can review every little detail, but I'll trust that this has been tested locally.

**Comment by jlipps** [comment]
thanks @dpgraham @KazuCocoa ! it is a big PR and we may find some issues as we continue to test, i did as much locally as i could think of doing. now we're in a good spot to move forward with new features and a much more lightweight app!

the only outstanding todo is the hazel server for the inspector. i have no idea how the updating process works, so @dpgraham i'd love your help with thinking through release and update when you have a bit of time. and maybe we can release a beta of each of these under the new versioning scheme to start looking for some feedback?

**Comment by KazuCocoa** [comment]
🎉

**Pull request merged**